src/core/router/history/base.js

```diff
@@ -42,8 +42,20 @@ export class History {
     const { config } = this;
     const base = this.getBasePath();
     const ext = typeof config.ext === 'string' ? config.ext : '.md';
+    const alias = config.alias || {};
+    const rules = Object.assign({}, alias);
+
+    // auto alias sidebar which is set to be absolute path
+    if (config.sidebarAbsolutePath) {
+      const sidebar = config.loadSidebar || '_sidebar.md';
```

Comment:
Do we need this `sidebarAbsolutePath`? Or can we just provide an absolute path in `loadSidebar`?

Reply:
Think about this: what happend if they set `sidebarAbsolutePath` to true, but it isn't an absolute path? Should it be an error? Will they be confused?

Here's an idea on how to detect absolute paths with a regex: https://stackoverflow.com/a/19709846/454780 (but we should allow paths starting with `/` to return true if we want.

Reply:
> Here's an idea on how to detect absolute paths with a regex: https://stackoverflow.com/a/19709846/454780 (but we should allow paths starting with `/` to return true if we want.

Yes, we should consider detecting too.